From 74edce484429249265baaee1e8a5d1785ee7afa7 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Tue, 7 Feb 2023 23:55:19 +0000
Subject: Remove spurious ssh-agent options

These cause regress/agent-getpeereid.sh to leave a stray ssh-agent
process lying around.

Bug: https://bugzilla.mindrot.org/show_bug.cgi?id=3536
Last-Update: 2023-02-07

Patch-Name: remove-spurious-ssh-agent-options.patch
---
 regress/agent-getpeereid.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/regress/agent-getpeereid.sh b/regress/agent-getpeereid.sh
index 79e9c7d9e..e0379445d 100644
--- a/regress/agent-getpeereid.sh
+++ b/regress/agent-getpeereid.sh
@@ -53,7 +53,7 @@ else
 	fi
 
 	trace "kill agent"
-	${SSHAGENT} -vvv -k >>$OBJ/ssh-agent.log 2>&1
+	${SSHAGENT} -k >>$OBJ/ssh-agent.log 2>&1
 fi
 
 rm -f ${OBJ}/agent
